apertium-eng.eng.rlx

```diff
@@ -329,6 +344,33 @@ SELECT V IF (-1 (prn pers)) ;
 SELECT SUB:1 Be IF (1 Ing) (0/* Have) ;
 SELECT Ger IF (-1/* Be) (1 To) ;
 
+SELECT Adv IF (0 Pr) (-1 Vblex) ;
+#Example (on):
+#He talked on.
+
+SELECT Cnjadv IF (0 Pr) (1 Det) (2 N) ;
+#Example (for):
+#Please get fish for the cat.
+
+SELECT Adv IF (0 N) (1 Supadj) ;
+#Example (smallest)
+#Canada is the second largest country in the world.
+
+SELECT Cnjadv IF (-1 Cm) (0 Pr) (1 Det) (2 N) ;
+#Example (for):
+#The cat didn't know, for the coconut didn't see.
+
+SELECT Cnjadv IF (-1 Cm) (0 Pr) (1 PrnSubj) ;
+#Example (for):
+#I was happy, for I had just won the lottery.
+
+SELECT Pr IF (0 Cnjadv) (1 PrnObj) ;
+#Example (for):
+#The food is for me.
+
+SELECT Pr IF (0 Cnjadv) (-2 Preadv) (-1 Adv) ;
```

Comment:
`Adv` or `Adj`?  ("as hot as...", etc.)

Reply:
But it looks like it could apply to the first one as well?

Reply:
`as` would have to be preceded by an adverb and a preadv.  I'll edit the rule such that:

SELECT Pr IF (0 Cnjadv) (-2 Preadv) (-1 Adv) (-3 Vbser)
`You are as quick as I am`

SELECT Pr IF (0 Cnjadv) (-2 Preadv) (-1 Adv) (-3 Vblex) (-4 Vbser)
`The Earth was formed as early as 4.54 billion years ago`


And I'll add a rule for each so that it affects adjectives, too, like "as hot as."

Reply:
If you're writing rules specifically for "as", then why don't you just put literal "as" in the rule?

Reply:
Sure.  I'll do that.